### ISSUE TITLE
[Feature] AddFeatureShortDescriptionHere

### ISSUE THREAD

**Issue by herrerodani** [NPBruce/valkyrie#1448]
It would be nice to have a global update button, so every scenario gets updated at once instead of having to update them one by one. Also, when an scenario gets updated, the list needs to be re-scrolled to find it again.

Thank you very much.

**Comment by vidimus** [comment]
Noe that Valkyrie has more than 100 scenarios, this feature seems to me as a must. Do you olan to add it in the new version, @mayjak ?

**Comment by BenRQ** [comment]
Please note that GitHub servers may not like so many fast requests coming from one IP to many repository.
Valkyrie app is not authenticated on GitHub so there are some specific restrictions.
If implemented, this should probably be tested with something like 1000 scenario, to be future proof :)

**Comment by vidimus** [comment]
> Please note that GitHub servers may not like so many fast requests coming from one IP to many repository.
> Valkyrie app is not authenticated on GitHub so there are some specific restrictions.
> If implemented, this should probably be tested with something like 1000 scenario, to be future proof :)

Oh, I see the point. Thanks for the explanation, @BenRQ 

At least, not refreshing the scenario list every time you update one scenario, that could save some time when a lot of scenarios have to been updated or downloaded. But maybe there's a reason for doing the refreshing, isn't it?

**Comment by BenRQ** [comment]
> At least, not refreshing the scenario list every time you update one scenario, that could save some time when a lot of scenarios have to been updated or downloaded. But maybe there's a reason for doing the refreshing, isn't it?

Reason is : I was lazy :)
Refreshing was just easier to implement.
I agree today it's problematic.

**Comment by mayjak** [comment]
@vidimus I agree it's a problem right now, but I'm not sure if it will make it to the nearest version. I'm trying to focus on Android + a list of smaller bugs first. But we'll see how it goes :)

**Comment by vidimus** [comment]
> @vidimus I agree it's a problem right now, but I'm not sure if it will make it to the nearest version. I'm trying to focus on Android (#1515) + a list of smaller bugs first. But we'll see how it goes :)

That's true. Compatibility with Android 11 is more necessary than this. Thanks to all for getting some free time to reply!
